services/wireguard/endpoint/kernelspace/client.go

```diff
@@ -132,3 +170,10 @@ func stringToKey(key string) (wgtypes.Key, error) {
 	}
 	return wgtypes.NewKey(k)
 }
+
+func suExec(args ...string) error {
```

Comment:
IMHO this suExec could be in some utils or some other generic package. This function can and should be reused along the linux (maybe unix) code.

Reply:
Moved it into `utils` package.